docs/users/configuration.md

```diff
@@ -0,0 +1,236 @@
+| `read-timeout`          	| Integer 	| 300000  	| This setting determines how long the proxy will wait to receive data from the server before timing out. If you use Forge, you may need to increase this setting.                                                                                                                              	|
+| `proxy-protocol`        	| Boolean 	| false   	| This setting determines whether or not Velocity should receive HAProxy PROXY messages. If you don't use HAProxy, leave this setting off.                                                                                                                                                      	|
+
+</div>
+
+##`query` section
+
+<div class="ox-hugo-table main-table">
+<div></div>
+
+| Setting name   	| Type    	| Default  	| Description                                                                                                              	|
+|----------------	|---------	|----------	|--------------------------------------------------------------------------------------------------------------------------	|
+| `enabled`      	| Boolean 	| `false`  	| Whether or not Velocity should reply to GameSpy 4 (Minecraft query protocol) requests. You can usually leave this false. 	|
+| `port`         	| Number  	| `25577`  	| Specifies which port that Velocity should listen on for GameSpy 4 (Minecraft query protocol) requests.                   	|
+| `map`          	| String  	| Velocity 	| Specifies the map name to be shown to clients.                                                                           	|
+| `show-plugins` 	| Boolean 	| false    	| Whether or not Velocity plugins are included in the query responses.                                                     	|
+
+</div>
+
+##`metrics` section
+
+<div class="ox-hugo-table main-table">
+<div></div>
+
+| Setting name  	| Type    	| Default                 	| Description                                                                                                 	|
+|---------------	|---------	|-------------------------	|-------------------------------------------------------------------------------------------------------------	|
+| `enabled`     	| Boolean 	| `true`                  	| Whether or not Velocity should send metrics to bStats.                                                      	|
+| `id`          	| UUID    	| Randomly generated UUID 	| A randomly generated UUID that uniquely identifies your Velocity server. You should not alter this setting. 	|
+| `log-failure` 	| Boolean 	| `false`                 	| Whether or not Velocity should log whenever it fails to connect to bStats.                                  	|
+
+</div>
+
+##The default configuration
+
+Below is the default configuration file for Velocity, `velocity.toml`.
+
+```plain
```

Comment:
Use toml for the language here

Reply:
fixed in 6605a4c52fe0d591205b8bd0268e19637160cde1